typescript/api/controllers/webservice/RecordController.ts

```diff
@@ -1189,7 +1194,11 @@ export module Controllers {
           }
           let result = await this.RecordsService.updateMeta(brand, oid, record, user);
```

Comment:
Should the result of the meta update be checked? I'm not sure if that will throw if it fails or not?

Reply:
It will throw an error if it fails which is caught and handled